service/api.go

```diff
@@ -531,24 +532,33 @@ func (as *APIServer) listVariants(w http.ResponseWriter, r *http.Request) {
 func (as *APIServer) validateProjectConfig(w http.ResponseWriter, r *http.Request) {
 	body := util.NewRequestReader(r)
 	defer body.Close()
-	yamlBytes, err := ioutil.ReadAll(body)
+	bytes, err := ioutil.ReadAll(body)
 	if err != nil {
 		gimlet.WriteJSONError(w, fmt.Sprintf("Error reading request body: %v", err))
 		return
 	}
 
+	input := validator.ValidationInput{}
```

Comment:
This seems like it's non-backward compatible since it's changing the server's expected payload, and I think some people might use outdated evergreen binaries to validate their files (not sure about this, but I seem to recall at one point that this was true).

Anyways, I don't think this needs to be changed, but might be worth looking out for in #evergreen-users.

Reply:
I think this is a good point; i hate dealing with backwards compatibility changes so I'm just going to bake it in https://github.com/evergreen-ci/evergreen/pull/4702